app/src/renderer/components/common/AppMenu.vue

```diff
@@ -2,6 +2,7 @@
   <menu class="app-menu">
     <div class="app-menu-main">
       <router-link
+        v-if="user.signedIn"
```

Comment:
i don't think we should be hiding these links. let's let the user visit the page and have a nice message asking them to sign in.